diff --git a/node_modules/debug/src/browser.js b/node_modules/debug/src/browser.js
index cd0fc35..b9dab0c 100644
--- a/node_modules/debug/src/browser.js
+++ b/node_modules/debug/src/browser.js
@@ -134,7 +134,9 @@ function useColors() {
 		// https://developer.mozilla.org/en-US/docs/Tools/Web_Console#Styling_messages
 		(typeof navigator !== 'undefined' && navigator.userAgent && navigator.userAgent.toLowerCase().match(/firefox\/(\d+)/) && parseInt(RegExp.$1, 10) >= 31) ||
 		// Double check webkit in userAgent just in case we are in a worker
-		(typeof navigator !== 'undefined' && navigator.userAgent && navigator.userAgent.toLowerCase().match(/applewebkit\/(\d+)/));
+		(typeof navigator !== 'undefined' && navigator.userAgent && navigator.userAgent.toLowerCase().match(/applewebkit\/(\d+)/)) ||
+		// Check if we're being used in ReactNative
+		(typeof navigator !== 'undefined' && navigator.product && navigator.product.toLowerCase() === 'reactnative');
 }
 
 /**
